libmscore/system.cpp

```diff
@@ -508,23 +508,33 @@ void System::layout2()
             return;
             }
 
+
```

Comment:
Not useful blank line ;-)

Reply:
So, you spotted it! Great :-)
But is's gone now.